### DISCUSSION TITLE
Undelete a user

### DISCUSSION THREAD

**Discussion by justdaj** [mastodon/mastodon#22718] [Q&A]
I have a user who deleted their account on my Instance, but now wants to reactivate it.  I can't see how to.

When I check the accounts under moderation, I can see the account is deleted but I have no options to do anything with the account.

Can it be completely removed from the system so that he can recreate it, or can it be reenabled?

I tried _tootctl accounts delete_ but it is still in my list of deleted users.  I also tried _tootctl accounts modify <name> --enable_ but it says there is no such user!

thanks

**Comment by rbairwell** [comment]
I don't believe this is possible: when a profile delete is made https://docs.joinmastodon.org/spec/activitypub/#supported-activities-for-profiles the local database erases the data - including posts, likes and follows - and then sends notifications to other servers to do likewise (removing the account from follow lists, deleting content etc).
Attempting to reactivate an account will cause "confusion" as some servers might not have marked the account as deleted when it it reactivated - causing it to be randomly deleted again in the future, links to posts have already been lost etc.
